testsuite/test_student_workflow.py

```diff
@@ -163,11 +163,11 @@ def test_user_cannot_review(client, STC, base_db_setup):
 
     review_message = "+10.0: Hacker"
     response = client.post(
```

Comment:
Q: Could you also put a test where the student comments with something like? :pray: 
```
+10.0: I really did a nice job!
```